agent/engine/docker_task_engine_linux.go

```diff
@@ -39,3 +44,23 @@ func (engine *DockerTaskEngine) updateTaskENIDependencies(task *apitask.Task) {
 func (engine *DockerTaskEngine) invokePluginsForContainer(task *apitask.Task, container *apicontainer.Container) error {
 	return nil
 }
+
+// updateCredentialSpecMapping is used to map the bind location of kerberos ticket to the target location on the application container
+func (engine *DockerTaskEngine) updateCredentialSpecMapping(desiredCredSpecInjection string, hostConfig *dockercontainer.HostConfig) {
+	// Inject containers' hostConfig.BindMount with the kerberos ticket location
+	bindMountKerberosTickets := desiredCredSpecInjection + ":" + defaultKerberosTicketBindPath + readOnly
+	if len(hostConfig.Binds) == 0 {
+		hostConfig.Binds = []string{bindMountKerberosTickets}
+	} else {
+		hostConfig.Binds = append(hostConfig.Binds, bindMountKerberosTickets)
+	}
+
+	if len(hostConfig.SecurityOpt) != 0 {
```

Comment:
does the agent populate `hostConfig.SecurityOpt`? i.e. will this code path be ever hit?